tests/kafkatest/tests/streams/streams_eos_test.py

```diff
@@ -160,10 +168,16 @@ def abort_streams(self, keep_alive_processor1, keep_alive_processor2, processor_
         self.wait_for_startup(monitor1, keep_alive_processor1)
 
     def wait_for_startup(self, monitor, processor):
-        self.wait_for(monitor, processor, "StateChange: REBALANCING -> RUNNING")
+        if self.group_protocol == "classic":
+            self.wait_for(monitor, processor, "StateChange: REBALANCING -> RUNNING")
+        else:
+            # In the streams group protocol, not all members will take part in the rebalance.
+            # We can indirectly observe the progress of the group by seeing the member epoch being bumped.
+            self.wait_for(monitor, processor, "MemberEpochBump")
         self.wait_for(monitor, processor, "processed [0-9]* records from topic")
 
-    def wait_for(self, monitor, processor, output):
+    @staticmethod
+    def wait_for(monitor, processor, output):
         monitor.wait_until(output,
-                           timeout_sec=480,
+                           timeout_sec=60,
```

Comment:
I agree that `480` seconds is a big time to wait are we sure 60 is enough to not have flaky results? Although I guess it really should be enough.

Reply:
Good catch. Did not want to commit that.